modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java

```diff
@@ -146,6 +159,11 @@
     /** CODEC_LIB_AVAILABLE */
     private static final boolean CODEC_LIB_AVAILABLE = PackageUtil.isCodecLibAvailable();
 
+    /** Registration of the JAI-EXT operations */
+    static {
+        JAIExt.initJAIEXT();
```

Comment:
Wondering, what if some code does not go though image worker or grid coverage processor, will we get JAI operations instead? If so, it's probably a good idea to tell people to call this one at the beginning of their program, and/or add it to GeoTools.init() and recommend calling that one.
Also, just to make sure, if JAIExt.initJAIEXT(); is called multiple times, the subsequent calls are going to relalize the init already completed, and it won't do anything, right?